src/Dynamic.js

```diff
@@ -8,7 +8,8 @@ class Dynamic extends React.Component {
         this.state = {
             barSize: "0%",
             dynamoVersion: "",
-            loadDescription: ""
+            loadDescription: "",
+            loadingTime: "Loading time: "
```

Comment:
Do we need to localize this the same as we did to other resources?

Reply:
No need, it's part of setting the properties of the bar. IE:
`window.setBarProperties('2.14','Launching..','100%', 'Loading time: 1222222100ms')`

Reply:
Gotcha